src/components/NewsAndArticles/index.jsx

```diff
@@ -0,0 +1,180 @@
+import React, { Component } from 'react';
+import './styles.css';
+import { charityAPI } from '../../clients';
+import Heading from '../Heading/index';
+
+class News extends Component {
+  state = {
+    data: {
+      heading: {},
+      description: '',
+      link: {},
+      articles: []
+    },
+    loading: false,
+    error: false,
+    errorMessage: ''
+  };
+
+  componentDidMount() {
+    this._getData();
+  }
+
+  _getData = () => {
+    this.setState({ loading: true, error: false });
```

Comment:
No, we need to 
1- set the `loading` to `true` since we are going to call API
2- set the `error` to `false` cause we don't know if the last request failed and the `error` is `true`